docs/StardustDocs/topics/schemasImportSqlGradle.md

```diff
@@ -0,0 +1,126 @@
+)
+
+import org.jetbrains.kotlinx.dataframe.annotations.ImportDataSchema
+```
+
+```kotlin
+const val URL = "jdbc:mariadb://localhost:3306/imdb"
+
+const val USER_NAME = "root"
+
+const val PASSWORD = "pass"
+
+const val TABLE_NAME = "actors"
+```
+To generate schema for the result of an SQL query,
+you need to define the SQL query itself
+and the same parameters to establish connection with the database.
+
+You should also specify the name of the generated Kotlin class
+as a first parameter of annotation `@file:ImportDataSchema`.
+
+```kotlin
+@file:ImportDataSchema(
+    "TarantinoFilmSchema",
+    URL,
+    jdbcOptions = JdbcOptions(USER_NAME, PASSWORD, sqlQuery = TARANTINO_FILMS_SQL_QUERY)
+)
+
+import org.jetbrains.kotlinx.dataframe.annotations.ImportDataSchema
+```
+
+```kotlin
+const val URL = "jdbc:mariadb://localhost:3306/imdb"
+
+const val USER_NAME = "root"
+
+const val PASSWORD = "pass"
+
+const val TARANTINO_FILMS_SQL_QUERY = "select name, year, rank,\n" +
+    "GROUP_CONCAT (genre) as \"genres\"\n" +
+    "from movies join movies_directors on  movie_id = movies.id\n" +
+    "     join directors on directors.id=director_id left join movies_genres on movies.id = movies_genres.movie_id \n" +
+    "where directors.first_name = \"Quentin\" and directors.last_name = \"Tarantino\"\n" +
+    "group by name, year, rank\n" +
+    "order by year"
+```
+
+### With Gradle Task 
+
+To generate schema for existing SQL table,
+you need to define a few parameters to establish JDBC connection:
+URL (passing to `data` field), username, and password.
+
+Also, the `tableName` parameter should be specified.
+
+```kotlin
+dataframes {
+    schema {
+        data = "jdbc:mariadb://localhost:3306/imdb"
+        name = "org.example.imdb.Actors"
+        jdbcOptions {
+            user = "root"
+            password = "pass" 
+            tableName = "actors"
+        }
+    }
+}
+```
+
+To generate schema for the result of an SQL query,
+you need to define the SQL query itself
+and the same parameters to establish connection with the database.
+
+```kotlin
+dataframes {
+    schema {
+        data = "jdbc:mariadb://localhost:3306/imdb"
+        name = "org.example.imdb.TarantinoFilms"
+        jdbcOptions {
+            user = "root" 
+            password = "pass"
+            sqlQuery = "select name, year, rank,\n" +
+                    "GROUP_CONCAT (genre) as \"genres\"\n" +
+                    "from movies join movies_directors on  movie_id = movies.id\n" +
+                    "     join directors on directors.id=director_id left join movies_genres on movies.id = movies_genres.movie_id \n" +
+                    "where directors.first_name = \"Quentin\" and directors.last_name = \"Tarantino\"\n" +
+                    "group by name, year, rank\n" +
+                    "order by year"
+        }
+    }
+}
+```
+
+After importing the data schema, you can now start to import any data from SQL table or as a result of an SQL query
```

Comment:
I'd remove "now", sounds weird behind "After doing this,..."

I'd specify what people should do with that schema. Because likely they'll need to read some data (adhering to the generated schema) and then `cast`/`convertTo` it to that schema. 

Actually, now that I think of it, for [OpenApi](https://kotlin.github.io/dataframe/schemasgradle.html#openapi-schemas) I generate companion helper functions to read a certain piece of JSON using that schema. Could be interesting to have for your schema importing as well :)